### PR TITLE
Adds many new species and their respective armour / items.

### PR DESCRIPTION
**Pull request by X0-11**
Species added:
Spartans
Jackals
Brutes (TODO)
Species icons modified:
Sangheili

Will probably have merge conflicts if /when pr #295 is merged.

### CODE REVIEW

File: code/modules/halo/weapons/covenant/ammo.dm [hunk 1]
```diff
@@ -6,11 +6,12 @@
 /obj/item/projectile/covenant/plasmapistol
 	damage = 25
 	accuracy = -1
-	icon = 'code/modules/halo/icons/Covenant Weapons.dmi'
-	icon_state = "PlasmapistolProj"
+	icon = 'code/modules/halo/icons/Covenant_Projectiles.dmi'
+	icon_state = "Plasmapistol Shot"
```
Comment (bloxgate) [inline code/modules/halo/weapons/covenant/ammo.dm]: Plasma pistol is two words

File: code/modules/halo/weapons/covenant/ammo.dm [hunk 2]
```diff
@@ -19,8 +20,8 @@
 /obj/item/projectile/covenant/plasmarifle
 	damage = 40 // more damage than MA5B.
 	accuracy = 1
-	icon = 'code/modules/halo/icons/Covenant Weapons.dmi'
-	icon_state = "PlasmarifleProj"
+	icon = 'code/modules/halo/icons/Covenant_Projectiles.dmi'
+	icon_state = "Plasmarifle Shot"
```
Comment (bloxgate) [inline code/modules/halo/weapons/covenant/ammo.dm]: Two words

**Comment by TheScreamingStar** [comment]
Later on I will finish up the rest on I will finish on the rest of the stuff for my sprites on this PR (higher rank armor, furred brutes, species aesthetics for them).

File: code/modules/halo/species_items/kig-yar.dm [hunk 3]
```diff
@@ -5,6 +5,51 @@
...
+	icon = 'code/modules/halo/icons/species/jackalclothing.dmi'
+	icon_state = "scouthelm"
+	sprite_sheets = list("Kig-Yar" = 'code/modules/halo/icons/species/jackalclothing.dmi')
+	armor = list(melee = 50, bullet = 15, laser = 50,energy = 10, bomb = 25, bio = 0, rad = 0)
+	species_restricted = list("Kig-Yar")
+	action_button_name = "Toggle Night Vision"
+	var/linked_glasses
+	var/mob/living/carbon/human/u
+
+/obj/item/clothing/head/helmet/kigyar/proc/tryequipNV()
```
Comment (bloxgate) [inline code/modules/halo/species_items/kig-yar.dm]: Could you put some underscores between the words in that proc name please?

File: code/modules/mob/living/carbon/human/species/outsider/brutes.dm [hunk 4]
```diff
@@ -0,0 +1,15 @@
+/datum/species/spartan
+	name = "Brute"
+	name_plural = "Brutes"
+	blurb = ""//Not in chargen
+	flesh_color = "#4A4A64"
+	blood_color = "#A10808"
+	icobase = 'code/modules/halo/icons/species/r_Augmented_Human.dmi' //The DMI needed modification to fit the usual format (see other species' dmis)
```
Comment (bloxgate) [inline code/modules/mob/living/carbon/human/species/outsider/brutes.dm:7]: This is temporary, I hope?

**Review by bloxgate — CHANGES_REQUESTED** [review]
Looking great so far XO! Just a couple questions though. Shouldn't the other covie species have some radiation resistance like the elites too? They are going to be using the same weapons after all.

Also, what does setting the species eyes to null do?

File: code/modules/mob/living/carbon/human/species/outsider/brutes.dm (hunk 4 above)
Comment (X0-11) [reply code/modules/mob/living/carbon/human/species/outsider/brutes.dm:7]: Yep. My copypasta for species has been revealed!

**Comment by X0-11** [comment]
eyes = null stops the species sprite assembler from drawing an eye overlay. Don't have one for those species yet.

**Comment by caelaislinn** [comment]
There's a couple of .tmp files there I'm not sure about, but this branch is getting deprecated shortly so we can just skip them when we come back to port them over.

**Comment by caelaislinn** [comment]
Manual merging this...

**Pull request closed without merging**